### PR TITLE
Remove old demo code

### PR DESCRIPTION
**Pull request by GrantMatejka**
These demos are hardcoded so we want to not include these, expecially since we are beginning integration

This should delete demo code and references to it all

### CODE REVIEW

File: apps/grant/src/index.js [hunk 1]
```diff
@@ -24,16 +24,12 @@ const vendorAppJson = {
...
-         },
-         "column1": {
-            "children": {
                "textFormField": {
                   "label": "Phone Number",
                   "validation": "[0-9]",
                },
                "button": {
-                  "action": "submit",
+                  "action": "Click Me!",
```
Comment (meinfr13nd) [inline apps/grant/src/index.js:32]: Is this just the text in the button? I thought that was separate from the action

File: src/components/form.js [hunk 2]
```diff
@@ -34,18 +35,22 @@ export default function formGenerator(props) {
         </div>
       );
     } else {
-      return information;
+      return <Button variant='outlined' onClick=
+        {() => {
+          alert('clicked');
+        }}>Click Me</Button>;
```
Comment (meinfr13nd) [inline src/components/form.js:41]: This looks good for a demo

**Review by meinfr13nd — APPROVED** [review]
I trust that you made the changes I commented on for a reason, otherwise everything looks good

File: apps/grant/src/index.js (hunk 1 above)
Comment (nauticalmonkey) [reply apps/grant/src/index.js:32]: I think submit is also still better than click me

**Pull request merged**